source/includes/common-mc-admin-config.rst

```diff
@@ -845,12 +845,34 @@ MinIO supports the following mechanisms:
 
 .. start-minio-notify-kafka-tls-client-auth
 
-Specify the client authentication policy of the Kafka broker(s). See
-`ClientAuthType <https://golang.org/pkg/crypto/tls/#ClientAuthType>`__ for 
-more information on possible values for this field.
+Specify the client authentication type of the Kafka broker(s).
+The following table lists the supported values and their mapping:
```

Comment:
> mappings

Should be plural, I think?

Reply:
...I don't know for certain, but lets do it anyways.

```suggestion
The following table lists the supported values and their mappings
```